internal/riverinternaltest/riverdrivertest/riverdrivertest.go

```diff
@@ -792,6 +795,88 @@ func ExerciseExecutorFull[TTx any](ctx context.Context, t *testing.T, driver riv
 			require.Equal(t, rivertype.JobStateCompleted, job.State)
 			require.Equal(t, []string{"tag"}, job.Tags)
 		})
+
+		// TODO(bgentry): these are probably in the wrong file or location within
+		// the file and should go somewhere else?
+		t.Run("JobFinalizedAtConstraint", func(t *testing.T) {
```

Comment:
RE TODO: If you feel it may still be wrong, should probably try to find a new convention for where to put it. Should probably find a way to resolve this TODO before merge since it's fairly trivial.

Reply:
Yes, would like to resolve this and other TODOs before merging. Left this note bc I didn't want to lose focus/flow on what I was doing at the time.

What do you think? Does it seem ok for this to live within `JobInsertFull`? The challenge is that we may not be able to control the params we need to using any other insert mechanism, so maybe it is in fact the most appropriate place?

Reply:
Yeah, we may want to consider a special "DB constraints" section at some point, but this seems fine for now.

Let's just drop the TODO and reorg later if it comes up.